### ISSUE TITLE
`CpuInfo` parser cannot handle ARM and PowerPC

### ISSUE THREAD

**Issue by skontar** [RedHatInsights/insights-core#1794]
PowerPC `cpuinfo` output looks like this:
```
processor	: 0
cpu		: POWER7 (architected), altivec supported
clock		: 3612.000000MHz
revision	: 2.0 (pvr 004a 0200)

processor	: 1
cpu		: POWER7 (architected), altivec supported
clock		: 3612.000000MHz
revision	: 2.0 (pvr 004a 0200)

processor	: 2
cpu		: POWER7 (architected), altivec supported
clock		: 3612.000000MHz
revision	: 2.0 (pvr 004a 0200)

processor	: 3
cpu		: POWER7 (architected), altivec supported
clock		: 3612.000000MHz
revision	: 2.0 (pvr 004a 0200)
```

ARM looks like this:
```
processor       : 0
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 38.40
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 1
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 38.40
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 2
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 38.40
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 3
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 38.40
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

Hardware        : BCM2709
Revision        : a01041
```

I am not sure if ARM needs to be supported but I think at least PowerPC is supposed to be supported.

**Comment by koalakangaroo** [comment]
@lonicerae - can you check this?

**Comment by lonicerae** [comment]
Originally the Cpuinfo parser was written against x86 architecture, need to enhance the parser or convert it into a combiner for other platforms.

**Comment by jsvob** [comment]
@lonicerae Hi, I see you have assigned this to yourself. Can I take this? I've done most of the work anyway already. If you want to finish it, I can at least send you what I have, just in case you find it useful.

**Comment by lonicerae** [comment]
Hi @jsvob , sorry that I have not got these kinds of machines at hand to do further investigation. Sure, please take it if you have enough information to work on this.

**Comment by jsvob** [comment]
https://github.com/RedHatInsights/insights-core/pull/1965

**Comment by miclark** [comment]
Fixed by #1965
